app/views/videos/_fields.html.erb

```diff
@@ -1,36 +0,0 @@
-<%= f.inputs do %>
```

Comment:
We removed this and then reinstated it...not sure why it's still showing as removed.  Will try again!